### PR TITLE
Provides a schema for the generated RDF.

### LINKED ISSUE #166: Provide schema for generated RDF
The generated output is well-defined, but not specified in a schema.

Such a schema is also needed to have the generated data be a valid RDF dataset.

### PR DESCRIPTION
**Pull request by fvalasiad**
Closes #166.

### CODE REVIEW

**Comment by zvr** [comment]
Thanks, but this is documentation, not a schema.

A schema is a machine-processable representation of the same.

**Comment by fvalasiad** [comment]
@zvr It's true I didn't fully understand what you meant by "schema" but still tried! Will look into it again with the new info.

**Comment by fvalasiad** [comment]
Will the machine processable representation be included in the same file or another newly created file? What would that be if that's the case.

I assume I also revert output.md to it's previous version, or is the updated documentation better?

**Comment by fvalasiad** [comment]
@zvr Is that good enough?

File: doc/build-recorder-schema.ttl [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2023 Fotios Valasiadis
+# Copyright (C) 2023 Alexios Zavras
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+@prefix b: <http://example.org/build-recorder#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Types
+b:process a rdfs:Class .
```
Comment (zvr) [inline doc/build-recorder-schema.ttl:9]: `rdfs` namespace definition is missing

File: doc/build-recorder-schema.ttl [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+b:file a rdfs:Class .
+
+# Process Properties
+b:pid a rdf:Property ;
```
Comment (zvr) [inline doc/build-recorder-schema.ttl:13]: `rdf` namespace?!? different one?

**Review by zvr — CHANGES_REQUESTED** [review]

File: doc/build-recorder-schema.ttl (hunk 1 above)
Comment (fvalasiad) [reply doc/build-recorder-schema.ttl:9]: Note to self: Don't open pull requests 5 am in the morning without proper sleep.

File: doc/build-recorder-schema.ttl (hunk 2 above)
Comment (fvalasiad) [reply doc/build-recorder-schema.ttl:13]: @zvr Well according to the results of my research while searching for how to solve this issue, I came upon this:

> The correct namespaces to include in the RDF/Turtle file are rdf and rdfs. The rdf namespace is used for the core RDF vocabulary, while the rdfs namespace is used for the RDF Schema vocabulary.

Is said information invalid? Do I just use rdfs everywhere?

**Comment by fvalasiad** [comment]
@zvr This one is ready for review.

**Comment by zvr** [comment]
Merging this, to be improved later.

**Pull request merged**